src/build.d

```diff
@@ -940,7 +940,13 @@ void processEnvironment()
     }
     if (env.getDefault("ENABLE_RELEASE", "0") != "0")
     {
-        dflags ~= ["-O", "-release", "-inline"];
+        dflags ~= ["-O", "-inline"];
+
+        /* This is what we really want, but it fails in auto-tester and semaphoreci
+         * because they don't use compilers that support the check switches.
+         * Can we fix this?
```

Comment:
Yes, you can add `-check=` and `-checkaction=` as accepted but ignored arguments in the gdmd script (although `-check=assert` is equivalent to `-fassert` in gdc).